### ISSUE TITLE
 "@types/node" dependency not aligned with Node V12 version

### ISSUE THREAD

**Issue by STDeslandes** [eclipse-theia/theia#8555]
### Bug Description:
The  "@types/node" dependency is not aligned with node version.
In fact I want to user worker_threads feature available in node V12 and import is not working due to this issue

### Steps to Reproduce:

1. add import { Worker } from 'worker_threads'; in any file of an extension
2. see that 'worker_threads' definition is not available

### Additional Information

- Operating System: Windows
- Theia Version: 0.0.0

**Comment by vince-fugnitto** [comment]
Just to mention, the `typings` are only useful when developing for this repository.
The typings do not affect downstream applications or extensions.

**Comment by STDeslandes** [comment]
> 
> 
> Just to mention, the `typings` are only useful when developing for this repository.
> The typings do not affect downstream applications or extensions.

I m developing my extension in the "packages" folder.

**Comment by vince-fugnitto** [comment]
> > Just to mention, the `typings` are only useful when developing for this repository.
> > The typings do not affect downstream applications or extensions.
> 
> I m developing my extension in the "packages" folder.

Will the extension be upstreamed as part of a pull-request for this repo?
If not, this should not be the approach when developing custom extensions for your application.

**Comment by STDeslandes** [comment]
> 
> 
> > > Just to mention, the `typings` are only useful when developing for this repository.
> > > The typings do not affect downstream applications or extensions.
> > 
> > 
> > I m developing my extension in the "packages" folder.
> 
> Will the extension be upstreamed as part of a pull-request for this repo?
> If not, this should not be the approach when developing custom extensions for your application.

I agree. I'm more testing possibilities and performances right now. But you are right.
